app/views/internal/classified_listings/index.html.erb

```diff
@@ -53,7 +54,95 @@
         </div>
       </div>
     </div>
+
+    <h3>Email Form:</h3>
+    <div class="email-area" style="text-align:left">
+      <h3>
+        Send to:
+        <br>
+        <%= email_field_tag :reporter_email_to, listing.user.email, class: "form-control", id: "reporter__emailto__#{listing.id}", required: true %>
+      </h3>
+      <h3>Subject:</h3>
+      <%= text_field_tag :reporter_email_subject, "Regarding your recent DEV Listing", class: "form-control", id: "reporter__subject__#{listing.id}" %>
+      <h3>Body:</h3>
+      <%= text_area_tag :reporter_email_body, listing.title, class: "form-control", style: "height: 200px;", id: "reporter__body__#{listing.id}" %>
+    </div>
+    <br>
+    <button class="btn btn-primary" type="button" id="send__email__btn__<%= listing.id %>">SEND EMAIL ✉️</button>
+    <div class="email__alert alert fade in" id="email__alert__<%= listing.id %>">
+    </div>
   </div>
+
+<script>
```

Comment:
I feel bad for a lot of embedded js. I would at least put into a separate file but it seems like we have this (anti)pattern all over internal  :sweat_smile:

Reply:
Yeah, I didn't want to stray away from the anti-pattern in internal for now. It should probably be refactored in the future though.